nc/source/physics/collision/Polytope.h

```diff
@@ -13,10 +13,13 @@ namespace nc::physics
             void Initialize(const Simplex& simplex);
             auto GetNormalData(size_t index) const -> NormalData;
             auto ComputeNormalData() -> size_t;
-            auto Expand(const Vector3& support, size_t* oldMinFace) -> bool;
+            auto Expand(const Vector3& supportCSO, const Vector3& worldSupportA, const Vector3& worldSupportB, const Vector3& localSupportA, const Vector3& localSupportB, size_t* oldMinFace) -> bool;
+            bool GetContacts(size_t minFace, Vector3* worldContactA, Vector3* worldContactB, Vector3* localContactA, Vector3* localContactB) const;
```

Comment:
I'd prefer to see a Contacts struct returned, rather than four out params

Reply:
Didn't ever consider that, fixed.